consensus/istanbul/backend/api.go

```diff
@@ -321,15 +458,55 @@ func (api *APIExtension) makeRPCBlockOutput(b *types.Block,
 		rpcTransactions[i] = klaytnApi.RpcOutputReceipt(tx, hash, head.Number.Uint64(), uint64(i), receipts[i])
 	}
 
-	r["committee"] = cInfo.committee
-	r["proposer"] = cInfo.proposer
-	r["round"] = cInfo.round
+	r["Committee"] = cInfo.committee
+	r["committeeFromExtra"] = cInfo.validatorsFromExtra
+	r["committeeSealFromExtra"] = cInfo.committeeFromExtraSeal
+	r["Proposer"] = cInfo.proposer
+	r["Round"] = cInfo.round
 	r["originProposer"] = cInfo.originProposer
+	r["roundProposer"] = cInfo.roundProposer
+	r["roundCommitte"] = cInfo.roundCommitte
 	r["transactions"] = rpcTransactions
 
 	return r
 }
 
+func (api *APIExtension) ValidateBlock(number *rpc.BlockNumber) (interface{}, error) {
```

Comment:
I think "block validation" usually means validating a block header including state root and its transactions. But, this method validates block extra data. How about changing its name to `ValidateBlockSeal`?